apps/rewards/src/vaults.ts

```diff
@@ -71,57 +72,57 @@ export const vaults: Partial<Record<Network, Array<Vault>>> = {
       address: '0x84e58d8faA4e3B74d55D9fc762230f15d95570B8',
       rewardTokens: [CRV, CVX, PYUSD],
       poolTokens: [PYUSD, USDC],
-      tokenWeights: [50, 50],
+      tokenWeights: [0, 100],
     },
     {
       address: '0xba4eb30f7F2e378249cf94E08F581e704326e9c6',
       rewardTokens: [CRV, CVX],
       poolTokens: [USDC, crvUSD],
-      tokenWeights: [50, 50],
+      tokenWeights: [100, 0],
     },
     {
       address: '0x86B222d44AC6cC56e75b3df01fdAD5Dc371EF538',
       rewardTokens: [CRV, CVX],
       poolTokens: [USDT, crvUSD],
-      tokenWeights: [50, 50],
+      tokenWeights: [100, 0],
     },
     {
       address: '0xeEB885Af7C8075Aa3b93e2F95E1c0bD51c758F91',
       rewardTokens: [BAL, AURA],
       poolTokens: [GHO, '0x8353157092ED8Be69a9DF8F95af097bbF33Cb2aF', USDC, USDT],
-      tokenWeights: [33, 0, 33, 33],
+      tokenWeights: [0, 0, 100, 0],
     },
     {
       address: '0x914255c0c289AEa36E378EBB5e28293b5ED278Ca',
       rewardTokens: [BAL, AURA],
       poolTokens: ['0x596192bB6e41802428Ac943D2f1476C1Af25CC0E', ezETH, ETH],
-      tokenWeights: [0, 50, 50],
+      tokenWeights: [0, 0, 100],
     },
     {
       address: '0x32D82A1C8618c7Be7Fe85B2F1C44357A871d52D1',
       rewardTokens: [BAL, AURA],
       poolTokens: ['0x05ff47AFADa98a98982113758878F9A8B9FddA0a', rETH, weETH],
-      tokenWeights: [0, 50, 50],
+      tokenWeights: [0, 0, 100],
```

Comment:
Isn't rETH more liquid?

Reply:
no

Reply:
yest it is, my brain read wsteth instead of weEth